janus.c

```diff
@@ -2895,6 +2896,8 @@ gint main(int argc, char *argv[])
 	core_limits.rlim_cur = core_limits.rlim_max = RLIM_INFINITY;
 	setrlimit(RLIMIT_CORE, &core_limits);
 
+        janus_log_init();
```

Comment:
I noticed later you correctly used tabs everywhere but this line (and the debug.h bit) :smile: